app/controllers/home_controller.rb

```diff
@@ -4,6 +4,12 @@ class HomeController < ApplicationController
   def index
     @q = User.ransack(params[:q])
     @users = @q.result(distinct: true).decorate
+    @user_list = User.all
```

Comment:
1. The name of the instance variable is misleading, it's not a list
2. Avoid using instance variables if you're not exposing the to the view / other methods.
3. Why not reuse what you have in the line six?